docker-compose.yml

```diff
@@ -33,6 +33,22 @@ services:
     networks:
       - default
 
+  nginx_goss:
```

Comment:
Goss is for testing images yeah? Should there be a test as well or is this for future testing?

Reply:
Yeah goss is for image testing - we can include a more substantive test for this particular feature though (something like the clamav tests).

In this container we mount a redirect file, make a HTTP request and verify the response code. Will update the PR.

Reply:
The idea of the nginx_goss containers is to make sure that during CI the `ONBUILD` step works.